### PR TITLE
fix(validation-i18n): prioritization of translated messages

### PR DESCRIPTION
**Pull request by Sayan751**
# Pull Request

### 🎫 Issues

Reported in https://discourse.aurelia.io/t/au2-validation-in-conditionally-rendered-component/5507/8.

### CODE REVIEW

File: packages/validation-i18n/src/localization.ts [hunk 1]
```diff
@@ -82,7 +84,7 @@ export class LocalizedValidationMessageProvider extends ValidationMessageProvide
       key ??= messageKey;
     }
 
-    return this.setMessage(rule, this.i18n.tr(this.getKey(key)));
+    return this.setMessage(rule, this.i18n.tr(hasTranslation ? i18nKey : this.getKey(key)));
```
Comment (bigopon) [inline packages/validation-i18n/src/localization.ts:87]: shouldn't we return early with `hasTranslation` instead? If the style of the package is to have a single return then all good.

File: .circleci/config.yml [hunk 2]
```diff
@@ -864,7 +864,7 @@ workflows:
       - e2e_test:
           <<: *filter_ignore_develop_release
           name: e2e_validation_issue_2025
-          path: "packages/__e2e__/11-validation-issue-2025"
+          path: "packages/__e2e__/11-validation"
```
Comment (bigopon) [inline .circleci/config.yml:867]: 👍

**Review by bigopon — APPROVED** [review]

File: packages/validation-i18n/src/localization.ts (hunk 1 above)
Comment (Sayan751) [reply packages/validation-i18n/src/localization.ts]: Done.

**Comment by bigopon** [comment]
nice work @Sayan751

**Pull request merged**